client/component/App.jsx

```diff
@@ -24,13 +45,17 @@ class App extends React.Component {
 
     render() {
         return (
-            <>
-            {Object.keys(this.state.product).length > 0 &&
-                <Price product={this.state.product}/> }
-            {Object.keys(this.state.product).length === 0 &&
-                <div>Something is wrong. Unless you paused as the page is rendering, this should never show up.</div>
-            }
-            </>
+            <WidgetArea>
+                {Object.keys(this.state.product).length > 0 &&
```

Comment:
i myself would create a renderItem function and handle the logic there, but this works

Reply:
I will consider doing this after I get my portion to MVP 👍